packages/shared/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.0.0",
   "private": true,
   "exports": {
-    "./logger": "./dist/logger/index.js",
```

Comment:
Ooof I didn't know about (or forgot about) this one

Reply:
Just checking, anything to do here?

Reply:
This was intentional. It fixed the weird `logger.logger.debug` change. My comment was to Mateusz; I just meant that I'd forgot it existed so I didn't think to try deleting it.